suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -121,3 +121,45 @@ tests:
       module: trash_restore.py
       name: Check trash restore functionality
       polarion-id: CEPH-83573298
+
+  - test:
+      config:
+        ec-pool-k-m: 2,1
+        ec_pool_config:
+          pool: rbd_pool
+          data_pool: rbd_ec_pool
+          ec_profile: rbd_ec_profile
+          image: rbd_image
+          snap: rbd_ecsnap
+          snap_rename: rbd_ecsnap_rename
+          clone: rbd_ecclone
+          size: 10G
+        rep_pool_config:
+          pool: rbd_rep_pool
+          image: rbd_rep_image
+          snap: rbd_snap
+          snap_rename: new_rbd_snap
+          clone: rbd_clone_image
+          size: 10G
+      desc: Rename image snapshots on an image on replicated and ecpools and its clones
+      module: rbd_snapshot_rename.py
+      name: rbd_snapshot_rename
```

Comment:
Can we keep test name more easily understood names other than module name,
i.e`test snapshot rename functionality` Please refer other tests.

Reply:
Done.